src/app/modules/players/players.view.html

```diff
@@ -1,20 +1,17 @@
 <div class="container-fluid text-center">
 
   <div class="col-xs-12">
-    <!-- TODO this string needs to be translated -->
-    <h2>seleziona giocatori</h2>
+    <h2 translate="players.SELECT_PLAYERS"></h2>
   </div>
 
   <div class="col-xs-12">
     <div ng-show="vm.savedPlayers.getAll().length">
-      <!-- TODO this string needs to be translated -->
-      sono stati selezionati {{ vm.selectedPlayers.getAll().length }} giocatori
+      <h4>{{'players.SELECTED_PLAYERS' | translate}}: {{vm.selectedPlayers.getAll().length}}</h4>
```

Comment:
I due punti fanno parte della lingua... sicuro che si faccia cosi?
E se in inglese volessi far vedere "12 is the magin number" come potrei fare?